app/pages/sections/supplier/select/ui.test.js

```diff
@@ -2,6 +2,7 @@ import nock from 'nock';
 import { ClientFunction, Selector } from 'testcafe';
 import { extractInnerText } from 'buying-catalogue-library';
 import content from './manifest.json';
+import { orderApiUrl } from '../../../../config';
 
 const pageUrl = 'http://localhost:1234/organisation/order-1/supplier/search/select';
```

Comment:
change `order-1` to `order-id`

Reply:
some of the ui-test are expecting the url to contain `order-1` they will need updating too